### ISSUE TITLE
Dadaset

### ISSUE THREAD

**Issue by LYF14020510036** [RexYing/diffpool#16]
Hello. Is REDDIT-MULTI-5K dataset part of REDDIT-MULTI-12K dataset?

**Comment by LYF14020510036** [comment]
Thank you very much for your reply. That is to say, there are 5+11 labels in the two datasets, right?

**Comment by RexYing** [comment]
Yes 5 classes for the 5K dataset; 11 classes for the 12K dataset. The original datasets are proposed by the Deep Graph Kernel Paper.

**Comment by LYF14020510036** [comment]
> Yes 5 classes for the 5K dataset; 11 classes for the 12K dataset. The original datasets are proposed by the Deep Graph Kernel Paper.

Thank you very much.

**Comment by LYF14020510036** [comment]
Hi. I want to ask you something about REDDIT-MULTI-12K dataset. The corresponding labels of AskReddit, AdviceAnimals, atheism, aww, IAmA, mildlyinteresting, Showerthoughts, videos, todayilearned, worldnews and TrollXChromosomes are 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 and 10 ??? I download the dataset from https://ls11-www.cs.tu-dortmund.de/staff/morris/graphkerneldatasets. Thank you very much.
